### PR TITLE
Bootstrap testing with nodejs

### PR DESCRIPTION
**Pull request by BafS**
This should be fix in the lib https://github.com/ProtonMail/sieve.js/compare/test-1?expand=1#diff-910eb6f57886ca16c136101fb1699231R11

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+[*]
+indent_size = 4
+indent_style = space
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```
Comment (KAYLukas) [inline .editorconfig]: Add trailing newline

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+node_modules
+*.log
```
Comment (KAYLukas) [inline .gitignore]: Add trailing newline

File: test/index.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import Sieve from '../sieve';
+import archive from './fixtures/archive';
+import folder from './fixtures/folder';
+
+test('archive to tree', t => {
+    const generatedTree = Sieve.toTree(archive.simple);
+    t.deepEqual(generatedTree, archive.tree);
+});
+
+// FIXME
```
Comment (KAYLukas) [inline test/index.js]: Is this failing because of
https://github.com/ProtonMail/sieve.js/issues/3 ?
If so, could you add that in the comment?

Comment (BafS) [reply test/index.js]: Now all the tests are failing, the field `Vacation` is not optional anymore ? Also there are nulls generated in the tree "List".

**Pull request merged**